content/post/2023/captioning-tools-extravaganza.md

```diff
@@ -0,0 +1,142 @@
+---
+title: "Captioning Tools Extravaganza!"
+date: 2023-10-09
+tags: [captions, ml]
+---
+
+In this post, I'm going to cover a wide range of automatic captioning tools based on my personal experiences with them. This is not a comprehensive list, I know there's more of them out there. For context, I'm deaf, and I'm not fluent in American Sign Language (ASL). Since 2018 when they started to become available, I've relied heavily on automatic captioning tools in my work and personal lives.
+
+By sharing my experiences in this post, I hope to help individuals, employers, conference organizers, live streamers, speakers, and anyone else interested get a better understanding of what tools are available and how well they work. I also want to share this information with any of my Deaf/HoH peers who are researching these tools as well.
```

Comment:
I think we need to spell hard of hearing out since not everyone will be familiar with that acronym. 

```suggestion
By sharing my experiences in this post, I hope to help individuals, employers, conference organizers, live streamers, speakers, and anyone interested in better understanding what tools are available and how well they work. I also want to share this information with any of my deaf and hard of hearing peers researching these tools.
```